sklearn/kernel_approximation.py

```diff
@@ -324,6 +324,9 @@ class RBFSampler(_ClassNamePrefixFeaturesOutMixin, TransformerMixin, BaseEstimat
         "random_state": ["random_state"],
     }
 
+    def _more_tags(self):
+        return {"preserves_dtype": [np.float64, np.float32]}
+
```

Comment:
nitpick: we tend to put that at the bottom of the class

Reply:
Done.

@glemaitre could you update https://github.com/scikit-learn/scikit-learn/issues/11000#issue-316232750. That mentions that you should put `_more_tags` just before `__init__`.